liblouis/compileTranslationTable.c

```diff
@@ -1400,6 +1402,11 @@ _lou_extParseChars(const char *inString, widechar *outString) {
 	for (k = 0; inString[k] && k < MAXSTRING - 1; k++) wideIn.chars[k] = inString[k];
 	wideIn.chars[k] = 0;
 	wideIn.length = k;
+	/* Fix issue #1535.
+	   For this function, the functional nature should be maintained,
+	   i.e. the same input -> the same behavior.
+	   Therefore, `errorCount` should not be externally influenced. */
+	errorCount = 0;
```

Comment:
I totally agree with the comment, but I wonder if setting `errorcount` to 0 is badly interacting with other code. Wouldn't it be better to store the old value locally and restoring it after?

Something along the lines of 

``` C
	/* Fix issue #1535.
	   For this function, the functional nature should be maintained,
	   i.e. the same input -> the same behavior.
	   Therefore, `errorCount` should not be externally influenced. */
        int oldErrorCount = errorCount;
	errorCount = 0;
        parseChars(NULL, &result, &wideIn);
	if (errorCount) {
		errorCount = 0;
		return 0;
	}
       errorCount = oldErrorCount;
```

Reply:
Reasonable.

Reply:
Reasonable. I have pushed a new commit about this.